pages/common/p4.md

```diff
@@ -0,0 +1,23 @@
+# p4
+
+> Version Control System.
+
+- Log in to the Perforce service:
+
+`p4 login`
+
+- Open a new file to add it to the depot:
+
+`p4 add`
+
+- Create or edit changelist description:
+
+`p4 change`
+
+- Open a file to edit:
+
+`p4 edit -c <change-number> <file-name>`
```

Comment:
Please use the token syntax(`{{}}`) and snake_case for the variables. Also, how does a user get the change_number ?

Reply:
I have modified my fork.
p4 change command will generate the change-number